src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkConfig.java

```diff
@@ -159,7 +165,7 @@ private void validateColumnForPartition(@Nullable String columnName, @Nullable S
     Schema.Field field = schema.getField(columnName);
     if (field == null) {
       throw new InvalidConfigPropertyException(
-        String.format("Partition column '%s' is missing from the table schema", columnName), "Partition Field");
+        String.format("Partition column '%s' is missing from the table schema", columnName), "partitionByField");
```

Comment:
can we use a constant for partitionByField instead of duplicating it here and in @Name tag.

Reply:
Done.